### DISCUSSION TITLE
Access logger in custom API extensions

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#4516] [Feature Requests]
It would be nice to be able to access the `logger` when writing custom API endpoints or hooks. This way, I won't have to rely on `console.log`.

**Comment by rijkvanzanten** [comment]
Wanna open a PR? Should be as simple as adding the logger [here](https://github.com/directus/directus/blob/main/api/src/extensions.ts#L96) and [here](https://github.com/directus/directus/blob/main/api/src/extensions.ts#L129) 🙂
